DEVELOPMENT.md

```diff
@@ -85,13 +99,34 @@ Follow the steps below to setup your development environment, replacing `<sgx|vi
    Note this command should not be used for a production instance, as it will leave the service
    open to all.
 
-### Development Docker image
+## Configuring
 
-For more details refer to [docker/README.md](./docker/README.md).
+The application expects the configuration to be submitted via the CCF proposals, for that you could use the CLI.
```

Comment:
nit, feel free to ignore: I'd add a link to the `docs/configuration.md` page here to allow the reader to get more details on the SCITT configuration (e.g., "for more details on configuring a SCITT application, please refer to this page").

Reply:
linked 3 mentions of configuration to the configuration doc